### ISSUE TITLE
text-transform is incorrectly detected as a `transforms2d` feature

### ISSUE THREAD

**Issue by borodean** [ismay/stylelint-no-unsupported-browser-features#146]
The following CSS:

```
.test {
  text-transform: uppercase;
}
```

Yields the `Unexpected browser feature "transforms2d" is not supported by ...` warning when the linter is configured to disallow the `transforms2d` feature.

**Comment by ismay** [comment]
See my comment here: https://github.com/ismay/stylelint-no-unsupported-browser-features/issues/137#issuecomment-766665032. Please open an issue in the doiuse repo. Thanks!
